### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by matviichuk92**

### CODE REVIEW

File: src/main/java/core/basesyntax/MainApp.java [hunk 1]
```diff
@@ -12,5 +12,17 @@
  * - Отправить все машини на работу и остановить работу всех машин.
  */
 public class MainApp {
+    public static void main(String[] args) {
+        Bulldozer yellow = new Bulldozer();
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/MainApp.java:16]: be attentive with [checklist](https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_02.html)
- Use abstract class to create Track, Bulldozer and Excavator instances.

Example:
Animal cat = new Cat();

Comment (Sofasmile) [inline src/main/java/core/basesyntax/MainApp.java:16]:
```suggestion
        Bulldozer yellowBulldozer = new Bulldozer();
```

File: src/main/java/core/basesyntax/MainApp.java [hunk 2]
```diff
@@ -12,5 +12,17 @@
...
+        yellow.doWork();
+        yellow.stopWork();
 
+        Excavator green = new Excavator();
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/MainApp.java:20]:
```suggestion
        Excavator green Excavator = new Excavator();
```

File: src/main/java/core/basesyntax/MainApp.java [hunk 3]
```diff
@@ -12,5 +12,17 @@
...
+        green.doWork();
+        green.stopWork();
+
+        Track orange = new Track();
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/MainApp.java:24]:
```suggestion
        Track orangeTrack = new Track();
```

**Comment by matviichuk92** [comment]
done

**Pull request closed without merging**